doc/classes/TextureRect.xml

```diff
@@ -56,29 +56,36 @@
 	</methods>
 	<members>
 		<member name="expand" type="bool" setter="set_expand" getter="has_expand">
-			If [code]true[/code] texture will expand to fit. Default value: [code]false[/code].
+			If [code]true[/code], the texture scales to fit its bounding rectangle. Default: [code]false[/code].
```

Comment:
>Default:

If 'value' should be removed, please update the guideline.

Reply:
https://github.com/GDquest/godot-docs/blame/24003933919f06c671e27aa01b6f93eea55b91d0/community/contributing/docs_writing_guidelines.rst#L317

Reply:
There's no guideline for "Default value", only an example I pulled from the existing docs. Is the default value worth mentioning at all in every case?

Reply:
IMO default value is quite useful with obscure discreet value like enum and boolean. Not so much for continuous value. I'm just not sure if I should use `Default value:` or `Default:`.